src/pages/case-studies/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { loadMdxFile, getI18nContentFilePaths } from 'modules/I18n/hoc/withMdx';
+
+import Container from 'modules/Common/containers/Container';
+import type { GetStaticProps } from 'next';
+import Hero from 'modules/Common/components/Hero';
+import Layout from 'modules/Common/containers/Layout';
+import { Link } from 'modules/I18n';
+import React from 'react';
+import TextContent from 'modules/Common/components/TextContent';
+import dayjs from 'dayjs';
+import { useRouter } from 'next/router';
+import useTranslation from 'next-translate/useTranslation';
+
+interface CaseStudy {
+  uri: string;
```

Comment:
I believe this is a URL more than a [URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier), since we use it in `href`.
```suggestion
  url: string;
```